uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -12,6 +16,28 @@ function tasksassignments_civicrm_config(&$config) {
 }
 
 /**
+ * @param array $wrappers
```

Comment:
You mean to use "Implementation of.." instead of "Implements.."?

If I use that I get warnings from civilint:


> Format should be "* Implements hook_foo().", "* Implements hook_foo_BAR_ID_bar() for xyz_bar().",, "* Implements hook_foo_BAR_ID_bar() for xyz-bar.html.twig.", or "* Implements hook_foo_BAR_ID_bar() for xyz-bar.tpl.php.".